### PR TITLE
01 creation pokemon list

### PR DESCRIPTION
**Pull request by bdtkevin**
Adding the components PokemonList & Pokemon for display the Pokemon list.
Call of the API to retrieve the list of Pokemon dynamically.

### CODE REVIEW

**Review by RobinDela — DISMISSED** [review]
No problem

**Review by RobinDela — DISMISSED** [review]
No problem

File: src/components/PokemonList.jsx [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+const PokemonList = () => {
+  /* Declaration of a variable with useState to store the return value of the
+    call to the API (useSate initialized with an empty array) */
+  const [pokemons, setPokemons] = useState([]);
+
+  /* Call the API to get the list of pokémons that we want to display on the
+    Homepage (here the 50 first Pokemon) */
+  useEffect(() => {
+    axios
+      .get('https://pokeapi.co/api/v2/pokemon?limit=20&offset=0')
```
Comment (LoicBrassart) [inline src/components/PokemonList.jsx]: The comment says 50 Pokemon ; the code shows it's limited to the first 20 results

File: src/components/css/Pokemon.scss [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+      color: #1B1B1B;
+      margin: 0;
+    }
+
+    p {
+      font-weight: 600;
+      font-size: 18px;
+      color: #fff;
+      margin: 0;
+      // display: inline;
```
Comment (LoicBrassart) [inline src/components/css/Pokemon.scss]: Dead code !

**Review by LoicBrassart — CHANGES_REQUESTED** [review]
Evils ! Eviiiils !

**Review by RobinDela — APPROVED** [review]
Clean

**Review by LoicBrassart — APPROVED** [review]
No More Evil !

**Pull request merged**